pkg/ebpf/c/tracer-ebpf.h

```diff
@@ -67,6 +67,20 @@ typedef struct {
     tcp_stats_t tcp_stats;
 } tcp_conn_t;
 
+#ifndef TCP_CLOSED_BATCH_SIZE
+#define TCP_CLOSED_BATCH_SIZE 3
```

Comment:
how did you decide on 3?

Reply:
It's the largest number before we blow up the eBPF stack of 512 bytes 😬 
I think we can batch more aggressively if change a bit of the code upstream, but I'd leave that for another PR

Reply:
Ah that makes sense, it would be to comment as much